### DISCUSSION TITLE
Shortcut being synced to OneDrive

### DISCUSSION THREAD

**Discussion by bsabrunosouza** [texstudio-org/texstudio#1662] [General]
My desktop is set up in OneDrive to be uploaded to the cloud. However, the TexStudio 3.1.2 shortcut is being synced (.ink extension). I leave it here for you to review the code so that such a problem does not occur.

**Comment by Gomesz785** [comment]
> TexStudio 3.1.2 shortcut is being synced (.ink extension)

so can you see the shortcut in the desktop? : attach a screenshot.
